### DISCUSSION TITLE
what is the proper procedure to shutdown an okd cluster with rook

### DISCUSSION THREAD

**Discussion by johnlongo** [rook/rook#8294] [Q&A]
Running OKD 4.7 and had an issue shutting down the cluster where the worker nodes would not stop. My current shutdown procedure is a three step process. 1) cordon nodes 2) issue shutdown 3) uncordon. this worked well before I installed rook now the worker nodes don't shutdown. Any advice will be greatly appreciated.

**Comment by leseb** [comment]
Can you clarify "worker nodes would not stop"?
